src/components/Avatar/Avatar.tsx

```diff
@@ -175,6 +169,19 @@ const AvatarStatusItems: FC<BaseAvatarProps> = React.forwardRef(
                 ? { 'aria-label': statusItemProps.ariaLabel }
                 : {})}
             >
+              {showStatusItemText ? (
+                <span
```

Comment:
You may use `getComputedStyle` of the span element by ref, something like:
```
const styles = getComputedStyle(ref.current);
console.log(styles.fontSize);
```

Create a ref for that span, get the computed style's `fontSize` and you should be good to go. Prob only need to do that in a `useMemo` as we care when the avatar is rendered.

In the memoized block, once you have that fontSize string, you may then `parseInt(n, 10)`, to convert that value to a number for the compare. Use the compare to update the `useState` boolean to either display or not display the count text via `showStatusItemText`.

Reply:
That way you may check if the font size is below 12px, adhere to the design system and hide the text when it's too small to be read.

Reply:
Please try and make these changes prior to merging, everything else LGTM.

Reply:
Done